.appveyor.yml

```diff
@@ -36,6 +40,9 @@ test: off
 matrix:
   fast_finish: true # stop all jobs if any job fails
   #allow_failures: # list jobs allowed to fail without stopping the rest
+  exclude:
+    - platform: x64 # do not build Win64 portable
```

Comment:
Is there a special reason to exclude a 64bit build?

Reply:
From what I understand, the portable version should run on the largest number of systems as possible, so the x86 version should be the right one.
A 64bit version should work, too, but the bat script generating it (i.e. the one I added to the "build" folder) should be modified, since I explicitly removed the 64bit option.

Reply:
fair enough

Reply:
With the updated script, a 64bit (local) build should also be possible, by compiling with msvc_build.bat after having defined the env variable BUILD_WIN_PORTABLE=ON